cnf-tests/hack/govet.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+. $(dirname "$0")/common.sh
```

Comment:
Would it make sense to have one single govet.sh file in the root `hack` folder instead of two?

Reply:
I'm thinking I might actually redistribute this code a bit - In order to have to not involve 'infra' every time we want to make a change within ztp, I was thinking I'd change the top-level `make job-ci` call `$(MAKE) -c ztp job-ci`, which can then be adapted to do 'go vet' and 'go test' and whatever other checks we want to add in the future.

How do you feel about that?